### DISCUSSION TITLE
Add "move window to other monitor" command to Window title bar menu

### DISCUSSION THREAD

**Discussion by bernd-wechner** [linuxmint/discussions#502] [Ideas]
One of my favourite features is the ability to move a window cleanly between two monitors. Alas I can only find this on the task bar right-click menu for a running app. Right clicking the title bar shows me a surprisingly similar menu but lacks that wonderful feature. 

That would be really useful because we often want to move a window to the other monitor but can't see the taskbar on this monitor ... (this is not a classic side by side monitors setup ;-). Just strikes me as a matter of menu consistency and completeness to have this command on the window title bar menu as well. 

Thoughts?

**Comment by cplittlfield** [comment]
'One of my favourite features is the ability to move a window cleanly between two monitors."

By golly, I didn't know that there was any way to do this, except with the keyboard shortcuts.  I'm all for having more ways of doing things.  Myself, I prefer keyboard shortcuts, but I can see the value of menu-driven ways, too.  Adding changing monitors to the title bar context menu makes a lot of sense.
